src/json-editor.component.scss

```diff
@@ -128,8 +131,8 @@ td.button-holder, th.button-holder {
   width: 46px;
 }
 
-div.error {
-  background: #f2968d;
+td.error {
+  background-color: #f2968d !important;
```

Comment:
No the problem is that somewhere the background color was set again... I am gonna check it again and see what causes it and fix it or let you know why important is needed.